samples/nrf9160/nrf_cloud_mqtt_multi_service/Kconfig

```diff
@@ -0,0 +1,71 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+menu "MQTT Multi Service Sample Settings"
+
+config POWER_SAVING_MODE_ENABLE
+	bool "Requests PSM from cellular network"
+
+config CLOUD_CONNECTION_RETRY_TIMEOUT_SECONDS
+	int "Sets the number of seconds between each cloud connection retry"
+	default 30
+
+config CLOUD_READY_TIMEOUT_SECONDS
+	int "Sets how long the sample will wait for nRF Cloud ready signal after connection"
+	default 30
+
+config LTE_INIT_RETRY_TIMEOUT_SECONDS
+	int "Sets the number of seconds between each LTE modem initialization retry"
+	default 30
+
+config AFTER_FOTA_LTE_SUCCESS_TIMEOUT_SECONDS
+	int "Sets the number of seconds to wait after a new firmware update before LTE connection is considered failed, and the firmware update is reverted."
+	default 300
+
+config DATE_TIME_ESTABLISHMENT_TIMEOUT_SECONDS
+	int "Sets the number of seconds to wait to determine the current time and date before giving up and moving on."
+	default 300
+
+config MAX_OUTGOING_MESSAGES
+	int "Sets the maximum number of device messages which may be enqueued before further messages are dropped"
+	default 64
+
+config APPLICATION_RESTART_TIMEOUT_SECONDS
+	int "Sets delay between each application soft restart in the event of failure"
+	default 30
+
+config APPLICATION_STACK_SIZE
+	int "Sets the stack size (in bytes) for the application thread of the sample"
+	default 2048
+
+config MIN_MSG_INTERVAL_SECONDS
+	int "Sets the minimum time to wait between sending MQTT messages (for instance, after connection reestablishment)"
+	default 1
+
+config SENSOR_SAMPLE_INTERVAL_SECONDS
+	int "Sets the time to wait between each sensor sample"
+	default 10
```

Comment:
I lean more towards 30 seconds to 1 minute, personally, since it's a demo and we don't want people to have to wait around for **_too long_** to see that it's working when they first try it. FWIW, the original ticket calls for 15 seconds. Can I talk you down to 30s?

Reply:
under a minute just seems like a waste of the user's SIM data.
i'll defer to @plskeggs since he wrote the ticket